src/DotNetWorker.Grpc/GrpcFunctionInvocation.cs

```diff
@@ -20,5 +21,7 @@ public GrpcFunctionInvocation(InvocationRequest invocationRequest)
         public override string FunctionId => _invocationRequest.FunctionId;
 
         public override TraceContext TraceContext { get; }
+
+        public RetryContext Context => _retryContext ??= new GrpcRetryContext(_invocationRequest.RetryContext);
```

Comment:
I think `_invocationRequest.RetryContext` can be null. From Alexey's PR to the node repo, there's a [comment here](https://github.com/Azure/azure-functions-nodejs-worker/blob/7005a1012e9ba88f6e402db36d3b20052b6a0a9a/src/public/Interfaces.ts#L186) that "The retry context of the current funciton execution. The retry context of the current function execution. Equals null if retry policy is not defined or it's the first function execution." I think maybe the property itself should be nullable or there should be some defaults?

Reply:
I'll apply defaults. It makes if for a pretty difficult API to work with if this expectation changes depending on which execution we're working with. Wish we had caught that in the original implementation as this should become available if requested with information that indicates that we're dealing with the first execution.

Reply:
Yeah - makes sense!! Retry count of 0 makes sense as a default but I'm having a hard time thinking of what the default would be for MaxRetryCount :/

Reply:
Yes, that will be an issue. I would really want to avoid releasing this with the null behavior, but without getting that information from the host, it will be difficult to expose the max count. I'll follow up to see what we can change in the host.

Reply:
Followed on the SDK item. I think we'll be going with a change there in order to properly enable this here, so this work will be blocked until we make that change (unfortunately). This capability is being positioned as preview in the SDK, but there isn't a great way to communicate that here beyond the approaches used in the SDK, which are not great.

Reply:
Would it make sense to check in this change with just the RetryCount property since we know we can default to "0"?

Reply:
We could, and I think there's some value even with the limited information. We made the change in the SDK last Friday, so though the behavior has been fixed there, it will take a few weeks until we can unblock this work (when it makes its way through deployment).

I'm on the fence. I'm all in favor of delivering incremental updates, but wonder if not having the max count for now will lead to some frustration with the initial implementation.